src/test/java/Homework19.java

```diff
@@ -0,0 +1,83 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+
+    @Test
+    public void deleteEmptyPlaylist() throws InterruptedException {
+        //navigatetoPage();
+        provideEmail("daria.pavlyuk@testpro.io");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        String playlistName = "Daria's playlist";
+        choosePlaylist(playlistName);
+        deletePlaylist();
+        getDeletePlaylistMessage();
+        assertDeletedPlaylistMessageIsDisplayed();
+
+
+
+
+    }
+    //step1
+    public void navigatetoPage()throws InterruptedException {
+        String url = baseUrl;
+        driver.get(baseUrl);
+        Thread.sleep(2000);
+    }
+        //step1
+        //public void navigatetoPage() {
+            //String url = "https://qa.koel.app/";
+            //driver.get(url);
+        //}
+
+        //step2
+        public void provideEmail (String email) throws InterruptedException{
+            WebElement emailField = driver.findElement(By.cssSelector("[type ='email']"));
+            emailField.click();
+            emailField.sendKeys(email);
+            Thread.sleep(2000);
```

Comment:
The usage of `Thread.sleep` in a method is not a good practice while writing an automation script. Instead of Thread.sleep please use the explicit wait method which is already defined in your BaseTest class.
` WebElement clickTestProPlaylist = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[type='password']")));`
`passwordField.click();`
 `passwordField.sendKeys(password);`